src/guides/configuration.md

```diff
@@ -0,0 +1,58 @@
+# Configuration
+
+Glimmer apps work right out of the box, but you can configure them to fit your needs.
+
+## Root Container
+
+Glimmer apps contain an `index.html` file that has the following div, which is where all your markup will reside in:
```

Comment:
I'm confused by this. Won't most of your markup actually reside in `src/ui/components`?